README.md

```
@@ -247,7 +247,124 @@ Gupshup is a messaging platform that enables bots and businesses to communicate
+ 1. Find your Postgres data directory:
+ ```bash
+ psql -U postgres -c "SHOW data_directory;"
+ ```
+
+ 2. Create SSL certificates using mkcert:
+ ```bash
+ mkcert -cert-file server.crt -key-file server.key postgres localhost 127.0.0.1 ::1
+ ```
+ In case you get the following error/warning:
+
+ ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)